README.md

```diff
@@ -129,6 +129,17 @@ setup_commands:
   - sudo /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-ctl -m ec2 -a stop
 ```
 
+### Unified CloudWatch Agent Images
+The AMI below is based on version 48 of the Ubuntu 18.04 AWS Deep Learning AMI and ships with Unified CloudWatch Agent v1.247348.0b251302.  
+Note that `T6Iq2faj` suffix in AMI Name serves as a unique identifier indicating Unified CloudWatch Agent is installed.  
+EC2 AMIs with Unified CloudWatch Agent pre-installed:
+
+| Base AMI                                     | AMI ID                | Region    |
```

Comment:
Should we add a new column showing the Unified CloudWatch Agent version installed on each AMI?

Reply:
Nit: shall we include corresponding ray version as well, or do the published AMIs always come with the ray wheels listed above?

Reply:
> Nit: shall we include corresponding ray version as well, or do the published AMIs always come with the ray wheels listed above?

Thanks Anna, ray is not installed on our CloudWatch agent AMIs. That way we do not have to worry about keep in sync with latest ray versions.